src/main/java/gmbs/model/Ticket.java

```diff
@@ -0,0 +1,63 @@
+package gmbs.model;
+
+import gmbs.model.generator.LottoGenerator;
+import gmbs.model.vo.LottoNumber;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class Ticket {
+
+    private final List<LottoNumber> numbers;
+
+    public Ticket(LottoGenerator generator) {
+        this.numbers = generator.getNumbers();
+    }
+
+    public boolean hasValue(LottoNumber value) {
```

Comment:
로또 번호를 인자로 받고, 로또 번호가 있는지 확인하는 메서드 이기 때문에,
네이밍이 구체적으로 변경되었으면 좋겠습니다.
```suggestion
    public boolean hasLottoNumber(LottoNumber value) {
```